Application/EdFi.Ods.Features/Container/Modules/ProfilesModule.cs

```diff
@@ -28,15 +30,27 @@ public override void ApplyConfigurationSpecificRegistrations(ContainerBuilder bu
                 .As<IAdminProfileNamesPublisher>()
                 .SingleInstance();
 
-            builder.RegisterType<ProfileResourceNamesProvider>()
+            builder.RegisterType<ProfileMetadataProvider>()
                 .As<IProfileResourceNamesProvider>()
```

Comment:
Should this interface registration entry be removed?
.As<IProfileResourceNamesProvider>()

builder.RegisterType<ProfileMetadataProvider>()
                .As<IProfileMetadataProvider>()
                .SingleInstance();